[docs] Improve information about manually uploading images in a private environment (#8735)
deckhouse/deckhouse: docs/documentation/pages/DECKHOUSE-FAQ.md

1. Pull Deckhouse images using the `d8 mirror pull` command.

   By default, `d8 mirror` pulls only the latest available patch versions for every actual Deckhouse release and the current set of officially supplied modules.
   For example, for Deckhouse 1.60, only one version, `1.60.4`, will be pulled, as it is sufficient to update Deckhouse from version `1.59`.

   The command below will pull Deckhouse tarballs for versions that are on the release channels (check [flow.deckhouse.io](https://flow.deckhouse.io) for the current status of the release channels):

   ```shell
   d8 mirror pull --source="registry.deckhouse.io/deckhouse/ee" --license="<DECKHOUSE_LICENSE_KEY>" $(pwd)/d8.tar
   ```

>    > Не забудьте изменить путь на соответствующий вашей редакции платформы – замените последнюю часть ссылки на `se` или `cse`. Например, для редакции SE ссылка будет выглядеть так: `registry.deckhouse.io/deckhouse/se`.

Reviewer: ```suggestion
   > Make sure to edit the path to match your platform edition — replace the last section of the link with `se` or `cse`. For example, for the SE edition, the link would look as follows: `registry.deckhouse.io/deckhouse/se`.
```

   For example, here is how you can pull all Deckhouse version images starting from version 1.45:

   ```shell
   d8 mirror pull --source="registry.deckhouse.io/deckhouse/ee" --license="<DECKHOUSE_LICENSE_KEY>" --min-version=1.45 $(pwd)/d8.tar
   ```

>    > Не забудьте изменить путь на соответствующий вашей редакции платформы – замените последнюю часть ссылки на `se` или `cse`. Например, для редакции SE ссылка будет выглядеть так: `registry.deckhouse.io/deckhouse/se`.

Reviewer: ```suggestion
   > Make sure to edit the path to match your platform edition — replace the last section of the link with `se` or `cse`. For example, for the SE edition, the link would look as follows: `registry.deckhouse.io/deckhouse/se`.
```

   ```shell
   d8 mirror push /tmp/d8-images/d8.tar "your.private.registry.com:5000/deckhouse/ee" --registry-login="<USER>" --registry-password="<PASSWORD>"
   ```

>    > Не забудьте изменить путь на соответствующий вашей редакции платформы – замените последнюю часть ссылки на `se` или `cse`. Например, для редакции SE ссылка будет выглядеть так: `your.private.registry.com:5000/deckhouse/se`.

Reviewer: ```suggestion
   > Make sure to edit the path to match your platform edition — replace the last section of the link with `se` or `cse`. For example, for the SE edition, the link would look as follows: `your.private.registry.com:5000/deckhouse/se`.
```

1. [Download and install the Deckhouse CLI tool](https://github.com/deckhouse/deckhouse-cli/blob/main/README.md#how-to-install).

> 1. Создайте строку аутентификации для `registry.deckhouse.io` командой:
>    ```shell     
>    YOUR_LICENSE_KEY="ваш_лицензионный_ключ" base64 -w0 <<EOF

Reviewer: ```suggestion
1. Create an authentication string for `registry.deckhouse.io` using the following command:
   ```shell     
   YOUR_LICENSE_KEY="your_license_key" base64 -w0 <<EOF
```

     {
       "auths": {
         "registry.deckhouse.io": {
           "auth": "$(echo -n license-token:${YOUR_LICENSE_KEY} | base64 -w0)"
         }
       }
     }
   EOF
   ```

>    > Не забудьте заменить `ваш_лицензионный_ключ` на свой реальный лицензионный ключ.
> 
> 1. Pull module images from their source registry, defined as a `ModuleSource` resource, into a dedicated directory using the command `d8 mirror modules pull`.

Reviewer: ```suggestion
   > Make sure to replace the `your_license_key` placeholder with your actual license key.

1. Pull module images from their source registry, defined as a `ModuleSource` resource, into a dedicated directory using the `d8 mirror modules pull` command.
```

   `d8 mirror modules pull` pulls only the module versions available in the module release channels at the time of copying unless the `--modules-filter` flag is set.

>    Пример файла `module_source.yml` с описанием ModuleSource:

Reviewer: ```suggestion
   Below is an example of a `module_source.yml` file with the ModuleSource definition:
```

   ```yaml
   apiVersion: deckhouse.io/v1alpha1
   kind: ModuleSource
   metadata:
     name: deckhouse
   spec:
     registry:

>        # Укажите строку, полученную в п.2 вместо CHANGE

Reviewer: ```suggestion
       # Replace CHANGE with the string produced in step 2
```

       dockerCfg: CHANGE
       repo: registry.deckhouse.io/deckhouse/ee/modules
       scheme: HTTPS

>      # Выберите подходящий канал обновлений: Alpha, Beta, EarlyAccess, Stable, RockSolid

Reviewer: ```suggestion
     # Select the appropriate release channel: Alpha, Beta, EarlyAccess, Stable, or RockSolid
```

    * Change the `.spec.registry.repo` field to the address that you specified in the `--registry` parameter when you uploaded the images;
    * Change the `.spec.registry.dockerCfg` field to a base64 string with the authorization data for your registry in `dockercfg` format. Refer to your registry's documentation for information on how to obtain this token.
 

>    Пример:

Reviewer: ```suggestion
   An example:
```

   ```yaml
   apiVersion: deckhouse.io/v1alpha1
   kind: ModuleSource
   metadata:
     name: deckhouse
   spec:
     registry:

>        # Укажите строку, полученную в п.2 вместо CHANGE

Reviewer: ```suggestion
       # Replace CHANGE with the string produced in step 2
```

       dockerCfg: CHANGE
       repo: registry.deckhouse.io/deckhouse/ee/modules
       scheme: HTTPS

>      # Выберите подходящий канал обновлений: Alpha, Beta, EarlyAccess, Stable, RockSolid

Reviewer: ```suggestion
     # Select the appropriate release channel: Alpha, Beta, EarlyAccess, Stable, or RockSolid
```

     releaseChannel: "Stable"
   ```

>    > Не забудьте изменить путь на соответствующий вашей редакции платформы – замените в ссылке `ee` на `se` или `cse`. Например, для редакции SE ссылка будет выглядеть так: `registry.deckhouse.io/deckhouse/se`.

Reviewer: ```suggestion
   > Make sure to edit the path to match your platform edition — replace the `ee` section of the link with `se` or `cse`. For example, for the SE edition, the link would look as follows: `registry.deckhouse.io/deckhouse/se/modules`.
```
